webapp/src/app.tsx

```diff
@@ -1754,6 +1762,11 @@ export class ProjectView extends data.Component<IAppProps, IAppState> {
                         </div> : undefined }
                     </div>
                 </div>
+                {!this.state.sideDocsLoadUrl && targetTheme && targetTheme.gettingStarted != undefined ? 
+                    <div id="getting-started-btn">
+                        <sui.Button class="bottom attached green" title={lf("Getting Started")} text={lf("Getting Started")} onClick={() => this.gettingStarted() } />
```

Comment:
button text and title should not be the same. Title can be more expressive and use the semantic UI css (see tooltip property)